shared/security/src/test/java/io/pravega/shared/security/utils/SecurityConfigDefaults.java

```diff
@@ -7,7 +7,7 @@
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
-package io.pravega.test.common;
+package io.pravega.shared.security.utils;
```

Comment:
It isn't clear looking at the package name or the class name (`io.pravega.shared.security.utils.SecurityConfigDefaults`) that this class is for use in test classes only, even though the location of the source file clarifies it a bit. I'd suggest that it should be made explicit by relocating this class to: `io.pravega.shared.security.test.SecurityConfigDefaults`.

Reply:
I have changed the package name